tools/wptrunner/wptrunner/wptmanifest/tests/test_tokenizer.py

```diff
@@ -1,22 +1,15 @@
-import sys
-import pytest
 import unittest
 
-from six.moves import cStringIO as StringIO
-
 from .. import parser
 from ..parser import token_types
 
-
-@pytest.mark.xfail(sys.version[0] == "3",
-                   reason="Tokenizer doesn't support py3")
 class TokenizerTest(unittest.TestCase):
     def setUp(self):
         self.tokenizer = parser.Tokenizer()
 
     def tokenize(self, input_str):
         rv = []
-        for item in self.tokenizer.tokenize(StringIO(input_str)):
+        for item in self.tokenizer.tokenize(input_str.encode("utf-8")):
```

Comment:
I'd rather we just made them all byte strings?

Reply:
Done